www/src/pages/components/progress.js

```diff
@@ -5,13 +5,13 @@ import LinkToSource from '../../components/LinkToSource';
 import PropTable from '../../components/PropTable';
 import ReactPlayground from '../../components/ReactPlayground';
 
-import ProgressBarBasic from '../../examples/ProgressBarBasic';
-import ProgressBarWithLabel from '../../examples/ProgressBarWithLabel';
-import ProgressBarScreenreaderLabel from '../../examples/ProgressBarScreenreaderLabel';
-import ProgressBarContextual from '../../examples/ProgressBarContextual';
-import ProgressBarStriped from '../../examples/ProgressBarStriped';
-import ProgressBarAnimated from '../../examples/ProgressBarAnimated';
-import ProgressBarStacked from '../../examples/ProgressBarStacked';
+import ProgressBarBasic from '../../examples/ProgressBar/ProgressBarBasic';
```

Comment:
lets remove the `ProgressBar` from the beginning of the file names since they are now namespaced in a directory 👍